src/PostCampaign.php

```diff
@@ -591,11 +596,11 @@ public static function get_footer_theme( array $css_vars ): array {
 		$default_footer_links_color = $css_vars['campaign_nav_color'] ? $css_vars['campaign_nav_color'] : '#1A1A1A';
 
 		if ( 'white' === $footer_theme ) {
-			$css_vars['footer_links_color'] = $css_vars['footer_links_color'] ? $css_vars['footer_links_color'] : $default_footer_links_color;
-			$css_vars['footer_color']       = '#FFFFFF';
+			$css_vars['footer_links_color']   = $css_vars['footer_links_color'] ? $css_vars['footer_links_color'] : $default_footer_links_color;
+			$css_vars['--footer--background'] = '#FFFFFF';
 		} elseif ( self::DEFAULT_NAVBAR_THEME === $css_vars['campaign_nav_type'] ) {
-			$css_vars['footer_links_color'] = null;
-			$css_vars['footer_color']       = null;
+			$css_vars['footer_links_color']   = null;
+			$css_vars['--footer--background'] = null;
```

Comment:
I had to increase Sonarcloud's duplicate string literal detection threshold to 5 (instead of default 3). Using a constant or variable like the rule suggests would further complicate this already complex code.